tests/runfiles/common.run

```diff
@@ -53,6 +53,24 @@ tags = ['functional', 'arc']
 tests = ['atime_001_pos', 'atime_002_neg', 'root_atime_off', 'root_atime_on']
 tags = ['functional', 'atime']
 
+[tests/functional/bclone]
+tests = ['bclone_crossfs_corner_cases_limited',
+    'bclone_crossfs_data',
+    'bclone_crossfs_embedded',
+    'bclone_crossfs_hole',
+    'bclone_diffprops_all',
+    'bclone_diffprops_checksum',
+    'bclone_diffprops_compress',
+    'bclone_diffprops_copies',
+    'bclone_diffprops_recordsize',
+    'bclone_prop_sync',
+    'bclone_samefs_corner_cases_limited',
+    'bclone_samefs_data',
+    'bclone_samefs_embedded',
+    'bclone_samefs_hole']
```

Comment:
It looks like a bunch of these new tests need to be added to PR commits.

There are also a bunch of existing tests in the `linux.run` file.  If possible it'd be good to adapt these so they can be run on FreeBSD as well.

```
[tests/functional/block_cloning:Linux]
tests = ['block_cloning_copyfilerange', 'block_cloning_copyfilerange_partial',
    'block_cloning_copyfilerange_fallback',
    'block_cloning_ficlone', 'block_cloning_ficlonerange',
    'block_cloning_ficlonerange_partial',
    'block_cloning_disabled_copyfilerange', 'block_cloning_disabled_ficlone',
    'block_cloning_disabled_ficlonerange',
    'block_cloning_copyfilerange_cross_dataset',
    'block_cloning_copyfilerange_fallback_same_txg']
tags = ['functional', 'block_cloning']
```